plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/lsp/AmazonQLspService.kt

```diff
@@ -241,7 +311,8 @@ private class AmazonQServerInstance(private val project: Project, private val cs
         get() = launcher.remoteEndpoint
 
     @Suppress("ForbiddenVoid")
-    private val launcherFuture: Future<Void>
+    var launcherFuture: Future<Void>
+        private set
```

Comment:
can still be val i think

Reply:
but then it needs to be a public val, is this what we want?